pom.xml

```diff
@@ -36,6 +36,8 @@
         <module>practitioner-service</module>
         <module>trial-config-service</module>
         <module>role-service</module>
+        <module>init-service</module>
```

Comment:
again about the name, I prefer to change it

redundant empty line

Reply:
remove redundant line